jvcl/run/JvDBCombobox.pas

```diff
@@ -143,6 +143,8 @@   TJvCustomDBComboBox = class(TJvCustomComboBox, IJvDataControl)
     procedure CMGetDataLink(var Msg: TMessage); message CM_GETDATALINK;
     procedure WMPaint(var Msg: TWMPaint); message WM_PAINT;
     procedure SetListSettings(const Value: TJvDBComboBoxListSettings);
+    procedure SetCaseSensitiveSearchValue(Value: Boolean);
+    function GetCaseSensitiveSearchValue: Boolean;
```

Comment:
Getters and Setters must be named after their associated property. In this case this means `SetCaseSensitiveValues` and `GetCaseSensitiveValues`